lib/concentrate/vehicle_position.ex

```diff
@@ -181,6 +181,23 @@ defmodule Concentrate.VehiclePosition do
       end
     end
 
+    defp overload_priority(block_id1, nil), do: block_id1
+
+    defp overload_priority(nil, block_id2), do: block_id2
+
+    defp overload_priority(block_id1, block_id2) do
+      cond do
+        String.contains?(block_id1, "-OL") ->
+          block_id1
+
+        String.contains?(block_id2, "-OL") ->
+          block_id2
+
+        true ->
+          first_value(block_id1, block_id2)
```

Comment:
```suggestion
          block_id1
```
`first_value()` checks for `nil`, but the function cases already check for `nil`, so this can just use the value directly.